generic/tests/kdump.py

```diff
@@ -69,7 +69,6 @@ def kdump_enable(vm, vm_name, crash_kernel_prob_cmd,
     kdump_config = vm.params.get("kdump_config")
     vmcore_path = vm.params.get("vmcore_path", "/var/crash")
     kdump_method = vm.params.get("kdump_method", "basic")
-    kdump_propagate_cmd = vm.params.get("kdump_propagate_cmd")
```

Comment:
why did you remove  kdump_propagate_cmd?

Reply:
kdump_propagate_cmd is kdumpctl propagate.
propagate:Helps to setup key authentication for ssh storage since it's impossible to use password authentication during
              kdump.
This key can be use only one time, after trigger kdump and system reboot the key can not be use in rhel7.5.
kdump.one_vm.with_ssh will trigger kdump more than once.